task_12.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""A really good docstring"""
+
+
+INTVAL = 1
+FLOATVAL = 0.1
+DECVAL = 0.1
+FRACVAL = 1/10
```

Comment:
`FRACVAL` is unfotunately not a fraction here nor is `DECVAL` a Decimal -- you need the `Decimal()` class, for example.